commit a251a6462125cfdb9448602fccefc1193757f335 (refs/changes/92/3992/3)
Author: Chris Burroughs <chris.burroughs@joyent.com>
Date:   2018-05-25T17:42:29+00:00 (1 year, 4 months ago)
    
    TRITON-398 upgrade sdcadm to use node v6
    Reviewed by: Pedro Palazón Candel <pedro@joyent.com>
    Reviewed by: Trent Mick <trentm@gmail.com>
    Approved by: Pedro Palazón Candel <pedro@joyent.com>
    Approved by: Trent Mick <trentm@gmail.com>

diff --git a/Makefile b/Makefile
index c901e2e..aa396a4 100644
--- a/Makefile
+++ b/Makefile
@@ -28,7 +28,7 @@ JSSTYLE_FLAGS	 = -f tools/jsstyle.conf
 CLEAN_FILES += ./node_modules ./build/sdcadm-*.sh ./build/sdcadm-*.imgmanifest ./build/shar-image ./man/man1/sdcadm.1 ./etc/sdcadm.completion
 
 
-NODE_PREBUILT_VERSION=v4.9.0
+NODE_PREBUILT_VERSION=v6.14.0
 ifeq ($(shell uname -s),SunOS)
 	NODE_PREBUILT_TAG=gz
 	# sdc-minimal-multiarch-lts 15.4.1
